tutorials/stats/4_global_services.py

```diff
@@ -0,0 +1,81 @@
+# %% [markdown]
+"""
+# 4. Global Services
+
+The following example demonstrates how to collect statistics
+from several global services.
+"""
+
+
+# %%
+import os
+import asyncio
+
+from dff.script import Context
+from dff.pipeline import Pipeline, ACTOR, ExtraHandlerRuntimeInfo, GlobalExtraHandlerType
+from dff.stats import StatsStorage, StatsExtractorPool, StatsRecord, default_extractor_pool
+from dff.stats.defaults import extract_timing_before, extract_timing_after
```

Comment:
Should we add them to `dff.stats.__init__`?

Reply:
Done